schemas/radiation.json

```diff
@@ -64,7 +64,7 @@
       }
     },
     {
-      "name": "application_form",
+      "name": "radiation_treatment_type",
```

Comment:
I still see the old codelist: 
![image](https://user-images.githubusercontent.com/15824252/82956369-a586ec00-9f7e-11ea-9d36-e19f94e1d155.png)

Do you need to merge master back?

Reply:
Yes I do. I know you updated this codelist earlier, so my branch must have been behind